week-1-practise/level-1/9-function-output-2.js

```diff
@@ -7,8 +7,8 @@ console.log(result);
 
 // Answer these questions:
 // - What does this program do? Run it and see, or Google some of the methods used ot understand them.
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
-// - What does the function return?
-// - How can this function be improved?
\ No newline at end of file
+// - How many parameters does the function take? two
+// - What is the function name? s
+// - Where is the function called? with what parameters? the function is called on line number 5 with the following parameters('code', 'your')
+// - What does the function return? the function return s
```

Comment:
`s` is the function name. What does that function give you?

Reply:
@martynamaron, The function will return code your

Reply:
Sure, but without the space in between :)

Reply:
Yeah, without the space